gr-digital/python/digital/bindings/constellation_soft_decoder_cf_python.cc

```diff
@@ -42,16 +43,16 @@ void bind_constellation_soft_decoder_cf(py::module& m)
              py::arg("constellation"),
              py::arg("npwr") = -1,
              D(constellation_soft_decoder_cf, make))
+
         .def("set_npwr",
              &constellation_soft_decoder_cf::set_npwr,
-             py::arg("npwr"),
+             py::arg("d_npwr"),
```

Comment:
The `.h` should be changed to `npwr` here, instead.

Reply:
Typo from previous rewrite. I saw that, but it didn't sink in. Good catch.